photosynthesis_metrics/fid.py

```diff
@@ -108,6 +85,26 @@ def forward(self, predicted_features: torch.Tensor, target_features: torch.Tenso
         Returns:
             score: Scalar value of the distance between image sets features.
         """
-        # Check inputs
-        super(FID, self).forward(predicted_features, target_features)
-        return self.compute(predicted_features, target_features)
+        super().forward(predicted_features, target_features)
```

Comment:
Is it redundant as the method use the functionality of parent class? The doc string could be moved to the class description. The same applies for `KID` class @ kid.py and `MSID` @ msid.py

Reply:
Agree

Reply:
I thought about moving this to parent class. 
Those metrics differ in shapes of feature vectors they can work on. KID needs the same sample size for both input and target features. MSID and FID can work even for different number of samples. It's important, because for those metrics we can compute target features on train and compare them with input features from validation, which  is usually much smaller.

Reply:
Could not agree with you more, the differences in processing are taken into account @ `compute_metric`. Therefore, the only new information/difference at `forward` is doc string.